scrapy/extensions/feedexport.py

```diff
@@ -169,6 +169,7 @@ def __init__(
         secret_key=None,
         acl=None,
         endpoint_url=None,
+        region_name=None,
         *,
```

Comment:
I wonder if it should be before or after *

Reply:
Let's put it after!

Reply:
Addressed in https://github.com/scrapy/scrapy/pull/5980/commits/e39fe1c9ef546b820317dbc275c730bbf4b8324f